package.json

```diff
@@ -147,7 +145,9 @@
         "classnames": "^2.3.1",
         "event-emitter-es6": "^1.1.5",
         "lodash.debounce": "^4.0.8",
-        "mobx": "^5.0.3",
+        "mobx": "^6.5.0",
+        "mobx-react-lite": "^3.4.0",
+        "mobx-undecorate": "^1.2.0",
```

Comment:
Is this a codemod? If so, can we move it to the devDependencies?

Reply:
@balakrishna-binary done